src/rgw/rgw_sal_motr.cc

```diff
@@ -1172,10 +1229,6 @@ void MotrStore::finalize(void)
   m0_client_fini(this->instance, true);
 }
 
-uint64_t MotrStore::get_new_req_id() {
```

Comment:
Why was it moved?

Reply:
It's actually there, line no. 4395. 
Cause or removing this and adding that- merging artifact.

Reply:
Yes, sir. Done.